### DISCUSSION TITLE
用的时间长了，似乎有点卡

### DISCUSSION THREAD

**Discussion by RainPPR** [Planshit/Tai#175] [General]
不知道是不是电脑配置的问题，感觉问题不大啊

每次打开总会卡个5秒左右

![image](https://user-images.githubusercontent.com/41764766/236678802-168789b4-a5a4-47fd-9895-562fe2ecada1.png)

是不是添加的分类太多了，有时还会卡十几秒就崩溃了

**Comment by RainPPR** [comment]
![image](https://user-images.githubusercontent.com/41764766/236678877-25e88b39-4af9-481f-9e5f-52badd8c3efe.png)

今年用了五百多个软件

**Comment by noberumotto** [comment]
收到反馈，我会尽快处理这个问题
